src/agent/adu_core_interface/src/adu_core_interface.c

```diff
@@ -505,6 +507,9 @@ void OrchestratorUpdateCallback(
     }
 
 done:
+    free(rootKeyPackageFilePath);
```

Comment:
calling free on a STRING_HANDLE that should be STRING_delete. Unless the calling function does something funky.